test/features/users.test.js

```diff
@@ -77,10 +71,11 @@ describe('Users', () => {
     const newUser = resLoggedIn.body.users[0];
     expect(resLoggedIn.jwt).toBe(undefined);
     expect(newUser.id).not.toBe(undefined);
-    expect(newUser.firstName).toEqual('Elowyn');
-    expect(newUser.lastName).toEqual('Platzer Bartel');
-    expect(newUser.email).toEqual('elowyn@example.com');
-    expect(newUser.birthYear).toEqual(2015);
+    expect(newUser.firstName).toMatch(/\w+(?:\d)/);
```

Comment:
Let me know if you think there's a better regex.

Reply:
Since you are comparing the endpoint to what has created in the database, you could just verify between the two, and validate the structure of the user in the users array, something like:

```js
expect(newUser).toEqual({
  id: user.id,
  first_name: user.firstName,
  ...
})
```

We aren't validating user properties, but the regex here insinuates that might be the case, so wouldn't use here.